src/apps/components/run.js

```diff
@@ -77,22 +86,27 @@ export class RunApp extends React.Component {
         this.props.dispatch(actions.newError(err));
     };
 
-    runView = () => {
+    runView = (options) => {
         let { dispatch } = this.props;
+        options = options || {};
 
         this.inputs.getValues()
         .then((values) => {
-            return this.view.run(this.props.bundle, values);
+            return this.view.run(this.props.bundle, values, options.debug);
```

Comment:
This sounds good to me. However since this touches juttle-client-library which now takes a debug flag in master branch I'd like to suggest that we do this particular change when we need a new option. Does that make sense?